### ISSUE TITLE
WebSockets fail

### ISSUE THREAD

**Issue by janno42** [fsr-itse/1327#589]
Rendering the preview of a document fails on production.

```
Internal Server Error: /documents/neue-seite-vom-2017-07-24-190658712247/render

KeyError at /documents/neue-seite-vom-2017-07-24-190658712247/render
'preview_b18140a1ba1c6d14322ca636319f73a1_10040'

Request Method: POST
Request URL: https://myhpi.de/documents/neue-seite-vom-2017-07-24-190658712247/render
Django Version: 1.11.3
Python Executable: /usr/bin/uwsgi-core
Python Version: 3.4.2
Python Path: [...]
Server time: Mo, 24 Jul 2017 19:07:10 +0200 Installed Applications:
['django_admin_bootstrapped',
 'django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'static_precompiler',
 'bootstrap3',
 'reversion',
 'guardian',
 'polymorphic',
 'django_extensions',
 'hijack',
 'compat',
 'channels',
 '_1327.main',
 '_1327.user_management',
 '_1327.documents',
 '_1327.information_pages',
 '_1327.minutes',
 '_1327.polls',
 '_1327.shortlinks',
 'django_auth_kerberos']
Installed Middleware:
['_1327.main.middleware.RedirectToNoSlash',
 'django.contrib.sessions.middleware.SessionMiddleware',
 'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 '_1327.user_management.middleware.IPRangeUserMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'django.middleware.locale.LocaleMiddleware',
 '_1327.user_management.middleware.LoginRedirectMiddleware']


Traceback:  

File "[...]/lib/python3.4/site-packages/django/core/handlers/exception.py" in inner
  41.             response = get_response(request)

File "[...]/lib/python3.4/site-packages/django/core/handlers/base.py" in _get_response
  187.                 response = self.process_exception_by_middleware(e, request)

File "[...]/lib/python3.4/site-packages/django/core/handlers/base.py" in _get_response
  185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "[...]/_1327/documents/views.py" in render_text
  241. 		'text': text

File "[...]/lib/python3.4/site-packages/channels/channel.py" in send
  88.             self.channel_layer.send_group(self.name, content)

File "[...]/lib/python3.4/site-packages/asgi_ipc/core.py" in send_group
  130.         for channel in self.group_channels(group):

File "[...]/lib/python3.4/site-packages/asgi_ipc/core.py" in group_channels
  137.         return self.group_store.flush_expired(group)

File "[...]/lib/python3.4/site-packages/asgi_ipc/store.py" in flush_expired
  188.                 for item, expiry in value[name].items()

Exception Type: KeyError at /documents/neue-seite-vom-2017-07-24-190658712247/render
Exception Value: 'preview_b18140a1ba1c6d14322ca636319f73a1_10040'
```

**Comment by Bartzi** [comment]
It seems to be a problem in `asg_ipc` but I'm not entirely sure. I found a possible fix and opened a [pull request ](https://github.com/django/asgi_ipc/pull/32), let's see what happens.

**Comment by Bartzi** [comment]
fixed with b657075ee60604e43a498d022d7cb1a4b61a981f
